pkg/cloud/openstack/clients/machineservice.go

```diff
@@ -51,6 +55,7 @@ import (
 
 const (
 	CloudsSecretKey = "clouds.yaml"
+	caFile          = "/etc/certs/cacert"
```

Comment:
when extending `cloud-config`, this would go away.

Reply:
Instead of caFile, I added CaSecretKey like this:
https://github.com/kubernetes-sigs/cluster-api-provider-openstack/commit/672c5affd15e14b1fb829e620087a8fe8c5c20de#diff-6f1595bb05179a80513b3baba49fbaf1R58